src/main/scala/firrtl/transforms/DedupAnnotations.scala

```diff
@@ -18,13 +18,16 @@ import firrtl.analyses.InstanceKeyGraph
 import scala.collection.mutable.ArrayBuffer
 
 object DedupAnnotationsTransform {
-
-  final class DifferingModuleAnnotationsException private (msg: String) extends PassException(msg)
-  object DifferingModuleAnnotationsException {
-    def apply(left: ReferenceTarget, right: ReferenceTarget): DifferingModuleAnnotationsException = {
-      val msg = s"${left.serialize} and ${right.serialize} have differing module binaries"
-      new DifferingModuleAnnotationsException(msg)
-    }
```

Comment:
I'm guessing this was unused? We need to also make this change to `master` then.

Reply:
Generally better to make follow on changes like this to master and backport separately.